src/main/java/model/Ladder.java

```diff
@@ -0,0 +1,33 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.path.PathGenerator;
+
+public class Ladder {
+    private final List<Line> lines;
+
+    private Ladder(final List<Line> lines) {
+        this.lines = lines;
+    }
+
+    public static Ladder from(final int height, final int personCount, final PathGenerator pathGenerator) {
+        validateHeight(height);
+        final List<Line> lines = new ArrayList<>();
+        for (int i = 0; i < height; i++) {
+            final Line line = new Line(pathGenerator.generate(personCount - 1));
```

Comment:
극한의 가독성을 추구한다면..!
`personCount - 1` 이 무엇인지 바로 이해할 수 있게 헬퍼 메서드를 만들어볼 수 있겠어요

```suggestion
            final Line line = new Line(pathGenerator.generate(pathCount(personCount)));
            
private static int pathCount(int personCount) {
    return personCount - 1;
}
```